tools-local/tasks/mobile.tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -257,7 +257,7 @@ private static string GetLatestBuildTools(string androidSdkDir)
         string? buildTools = Directory.GetDirectories(Path.Combine(androidSdkDir, "build-tools"))
             .Select(Path.GetFileName)
             .Where(file => !file!.Contains("-"))
-            .Select(file => Version.TryParse(Path.GetFileName(file), out Version? version) ? version : default)
+            .Select(file => { Version.TryParse(Path.GetFileName(file), out Version? version); return version; })
```

Comment:
```suggestion
            .Select(file => Version.TryParse(Path.GetFileName(file), out Version? version) ? version : null)
```

Reply:
Thanks but given this PR is needed to unblock arcade validation and this is just a stylistic change I'll skip the suggestion :)

Reply:
My goal here is to restore the code to essentially its original form while still working around the compiler issue.

Reply:
Yeah I'll keep it in mind for future changes to the file, right now I just want to get the builds unblocked.